.github/workflows/ci.yaml

```diff
@@ -622,14 +622,14 @@ jobs:
         run: |
           . venv/bin/activate
           python --version
-          pylint --ignore-missing-annotations=y homeassistant
+          pylint --ignore-missing-annotations=y homeassistant -d fixme
```

Comment:
These changes are not related to integration and should not be located here.

Reply:
Thanks for spotting! It wasn't meant to be included. Fixed.